student.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
```

Comment:
- [ ] Please make `classroom` an `attr_reader` because its a read-only in this case